packages/liveblocks-core/src/room.ts

```diff
@@ -3088,11 +3097,16 @@ export function createRoom<
       reconnect: () => managedSocket.reconnect(),
       disconnect: () => managedSocket.disconnect(),
       destroy: () => {
-        syncSourceForStorage.destroy();
```

Comment:
is syncSourceForStorage no longer needed here?

Reply:
Huh? Oops! Thanks for catching that, definitely unintended!